src/commands/pulls.js

```diff
@@ -0,0 +1,6 @@
+exports.command = 'pulls <command>'
+exports.desc = 'Manage GitHub pull requests'
+exports.builder = function (yargs) {
+	return yargs.commandDir('pulls')
+}
+exports.handler = function () { }
```

Comment:
See: https://github.com/yargs/yargs/blob/master/docs/advanced.md#commanddirdirectory-opts

Reply:
Add that as a healthy comment to the file.

Reply:
Done, cheers 👍